### ISSUE TITLE
Voice commands in Italian - Loading project

### ISSUE THREAD

**Issue by teguria** [IBVSoftware/CollaBoard-Feedback#322]
v 1.3.108
HP Spectre

Tested the voice command with CB in Italian. Spelled the project code in Italian and, instead of putting numbers, it recognizes phrases in english.

**Comment by corradocavalli** [comment]
Did you restart Collaboard after Change?



Da: alead<mailto:notifications@github.com>
Inviato: Tuesday, September 12, 2017 9:55 AM
A: IBVSoftware/CollaBoard-Feedback<mailto:CollaBoard-Feedback@noreply.github.com>
Cc: Subscribed<mailto:subscribed@noreply.github.com>
Oggetto: [IBVSoftware/CollaBoard-Feedback] Voice commands in Italian - Loading project (#322)


v 1.3.108
HP Spectre

Tested the voice command with CB in Italian. Spelled the project code in Italian and, instead of putting numbers, it recognizes phrases in english.

**Comment by teguria** [comment]
Yes I did

**Comment by corradocavalli** [comment]
Recognition of terms it depends on Cortana settings, that's why you don't see Italian terms but the sounding similar equivalent in English.

**Comment by teguria** [comment]
Is there anything we can do about it? I think most people would expect that, as long as you change CB language, then you can use the voice commands everywhere in CB, including inputting Key and code.

**Comment by corradocavalli** [comment]
It should be possible, I will investigate it.

On 13 Sep 2017, at 07:58, alead <notifications@github.com<mailto:notifications@github.com>> wrote:


Is there anything we can do about it? I think most people would expect that, as long as you change CB language, then you can use the voice commands everywhere in CB, including inputting Key and code.

**Comment by tamarawasser** [comment]
@teguria and @corradocavalli what's going on here guys? There is label "waiting confirmation". Do I have to test if it's working?

**Comment by corradocavalli** [comment]
@tamarawasser Yes, but you need an Italian accent and an Italian Win10 installation and Cortana in Italian, otherwise think it can be closed.

**Comment by tamarawasser** [comment]
@teguria please can you test and if it's working close this issue. Thanks

**Comment by gsantopaolo** [comment]
@teguria can you test it, @tamarawasser can you push @teguria to test please?

**Comment by teguria** [comment]
We need to set up a laptop with Windows 10 in Italian and Cortana in Italian. @mtg1986 do we have a laptop we can use? I gave my surface to Gianluca

**Comment by mtg1986** [comment]
@teguria : I would not do that. Just switch the language on the Surface Hub to Italian and test it there and then we can test on the Hub.

**Comment by teguria** [comment]
@mtg1986 it's the only way we can test this, otherwise if we just change language it won't be the same experience

**Comment by mtg1986** [comment]
@gsantopaolo : Should we invest time to test this again? I think the use case of people using voice commands for loading a project is very low prio and I would like to close this bug. What do you think?
